xlive/H2MOD/Modules/Shell/Config.cpp

```diff
@@ -721,17 +721,14 @@ void ReadH2Config() {
 				switch(std::stoi(ini.GetValue(H2ConfigVersionSection.c_str(), "experimental_rendering", "0")))
 				{
 					default:
+						//Incase any of the old rendering modes were used for a higher fps, set it back to 60.
+						H2Config_fps_limit = 60;
 					case 0:
 						H2Config_experimental_fps = _rendering_mode_none;
 						break;
```

Comment:
the statement inside of default should be in case 0: and both should execute it

Reply:
default falls through into case 0

Reply:
was under the impression that we were limiting it to 60 if we were using _rendering_mode_none, so we wouldn't have complaints about slowdown.
Clarified with kant and it's only for resetting the fps limit for people who had the other modes set.